gordon_janitor_gcp/reconciler.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Module to compare desired record sets produced from <TODO: generic name>
+and actual record sets from Google Cloud DNS, then publish corrective
+messages to the internal ``changes_channel`` if there are differences.
+
+This client makes use of the asynchronous DNS client as defined in
+:py:mod:`gordon_janitor_gcp.cloud_dns`, and therefore must use
+service account/JWT authentication (for now).
+
+.. attention::
+
+    Most likely, only the core janitor logic should interact with this
+    reconciler client, and won't be worked with directly.
```

Comment:
You interpreted it correctly. How can I word that better?

Reply:
It would be clearer to me if the last part (_, and won't be worked with directly._) wasn't there. But if you think this is clear the way it is, I'm fine with that, I don't trust my grammar parsing skills at all.